### DISCUSSION TITLE
Support onPaste event on input

### DISCUSSION THREAD

**Discussion by nick-keller** [JedWatson/react-select#5692] [Ideas]
# Idea
It would be nice to be able to pass a custom event hendler for the onPaste event on the input.

# Why?
For multiselect it would allow us to parse pasted value as simple tabs / new-lines array to handle pasting multiple values from Excel or from a list. This is impossible to do now since pasting on an input removes all new-lines and tabs characters.

If this idea sounds interesting I can make a PR

**Comment by raulfdm** [comment]
I have the same need. @nick-keller , have you find an alternative/workaround to implement this?

Giving the event on `onInputChange` would be do trick already
